doc/specs/stdlib_hash_procedures.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Hash procedures
+title: hash
```

Comment:
Actually I would prefer reading `Hash procedures` over `hash`, which is more informative to me.
E..g., `Numerical integration` can be easier understood than `quadrature`.

Reply:
Why don't we choose whether to list descriptive names or module names and use it consistently? I assumed we were listing module names. Can we merge this as is, and I'll open a new issue to discuss whether to switch this table to descriptive names?

Reply:
Yes, merge it as is. At least it is consistent now. Thank you.